ixmp/cli.py

```diff
@@ -140,9 +140,11 @@ def config(action, key, value):
 
 
 @main.command()
+@click.option('--max-row', type=int,
+              help='Max row numbers in each sheet.')
```

Comment:
What is the purpose of parameter?

Reply:
This is to let the input argument of max_row being passed to to_excel().

Reply:
I meant what is the need of max_row parameter in principle? do you want to limit a number of elements in parameter, e.g. save only parameters up to 500 elements - something like this?

Reply:
I don't think there's any harm in adding this!